pyper/sink/bigquery.py

```diff
@@ -0,0 +1,39 @@
+import json
+import typing as t
+
+from google.cloud import bigquery
+from pydantic import Field
+
+from pyper.sink import Sink, SinkConfig
+
+
+class BigquerySinkConfig(SinkConfig):
+    target_table: t.Union[str, bigquery.TableReference]
+    is_streaming: bool = Field(default=True)
+
+
+class TargetTableNotFound(Exception):
+    pass
+
+
+class BigQuerySink(Sink):
+    client: bigquery.Client
+    config: BigquerySinkConfig
+
+    def execute(self) -> None:
+        if not self.config.is_streaming:
```

Comment:
how about divide implementation of execute into streaming and batch?
pipe core maybe has streaming and batch processing method.